source/extensions/common/wasm/wasm.cc

```diff
@@ -930,7 +936,14 @@ Wasm::Wasm(absl::string_view vm, absl::string_view id) {
   wasm_vm_ = Common::Wasm::createWasmVm(vm);
   id_ = std::string(id);
   if (wasm_vm_) {
+
 #define _REGISTER(_fn) registerCallback(wasm_vm_.get(), #_fn, &_fn##Handler);
+    _REGISTER(getTotalMemory);
```

Comment:
Isn't 'getTotalMemory' removed or renamed to "getTotalMemoryHandler"?

Reply:
getTotalMemory is the name that is imported into the VM.  getTotalMemoryHandler is the code in envoy which handles that call-out.